adminCheck.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+require ('adminFunctions.php');
+
+$userName = sanitiseString($_POST['InputUserName']);
+$userPassword = sanitiseString($_POST['InputPswd']);
+
+session_start();
+
+$boolResult = pullAndComparePasswords ($db, $userPassword, $userName);
```

Comment:
PSR says `functionName($arg, $arg)` - no space before bracket
Mike says, `$boolResult` is a terrible variable name